src/components/MovieCard/MovieCard.scss

```diff
@@ -1 +1 @@
-// 
+//
```

Comment:
Let's add some styles for images to make them look better. Actually, only one property will be enough to fix it
![image](https://user-images.githubusercontent.com/96409563/233273712-342b6410-18eb-4f5d-b41b-95bd282ff50d.png)